docs/topics/templates.txt

```diff
@@ -390,6 +390,24 @@ Requires Jinja2_ to be installed:
 Set :setting:`BACKEND <TEMPLATES-BACKEND>` to
 ``'django.template.backends.jinja2.Jinja2'`` to configure a Jinja2_ engine.
 
+.. warning::
```

Comment:
I would prefer [adding a system check](https://github.com/django/django/blob/5d4d62bf4fe887fcd30f9f0449de07ae76ea5968/django/contrib/admin/checks.py#L29-L65) for this requirement so that the admin doesn't get "special privileges" in the templates documentation.